### DISCUSSION TITLE
Production Working install guide needed

### DISCUSSION THREAD

**Discussion by asimat** [cobbler/cobbler-web#41] [Q&A] [answered]
Is there a working installation guide for latest cobbler_web to run in production ? can be for centos/alma/rocky.

Development server works with localhost:4200, but only from the local machine. I cannot connect from another machine using IP.

If I add --prod then I get import errors, but without it in seems to have worked.

**Comment by SchoolGuy** [comment]
Okay so if this is about Cobbler Web, then I must sadly say that it is not ready for production yet. The current Cobbler release series 3.3.x does not have a web interface and due to resources this repository will take a lot of time to be ready. We are focusing our efforts towards the new CLI which can be used independently from the server.

**Comment by SchoolGuy** [comment, marked as answer]
Okay so I added a Dockerfile to this repository which is production ready. The image is not pushed yet to anywhere but I will make it available as soon as the basic CRUD functionality is available for the items and actions can be executed.

**Reply by asimat** [reply]
Thanks, looking forward to it.
